core/impl/src/test/resources/resources/base/components/charts/JFreeChartRender.xml

```diff
@@ -1,66 +0,0 @@
-<DesignerComponent>
```

Comment:
Why isn't this file being recognized as moved?

Reply:
Because we don't need so many duplicated component definition files to run the tests, so some were removed. Actually, I don't think we should be duplicating resources in the test folder, we should use the real ones, so maybe we can change this in future PRs.